spec/passaporte_web_spec.rb

```diff
@@ -16,16 +16,16 @@
         c.application_secret = 'some-app-secret'
         c.user_token = nil
       end
-      PassaporteWeb.configuration.url.should == 'http://some/where'
-      PassaporteWeb.configuration.user_agent.should == 'My App v1.0'
-      PassaporteWeb.configuration.application_token.should == 'some-app-token'
-      PassaporteWeb.configuration.application_secret.should == 'some-app-secret'
-      PassaporteWeb.configuration.user_token.should be_nil
+      expect(PassaporteWeb.configuration.url).to eq('http://some/where')
+      expect(PassaporteWeb.configuration.user_agent).to eq('My App v1.0')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.